### PR TITLE
Update vgan.py

### PR DESCRIPTION
**Pull request by Javihaus**
Include np.random.seed(2)

### CODE REVIEW

File: src/nbsynthetic/vgan.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 from keras.models import Sequential
 from tensorflow.keras.optimizers import Adam
 from tqdm import tqdm
+np.random.seed(2)
```
Comment (Javihaus) [inline src/nbsynthetic/vgan.py:20]: We needed to set a seed¡
